astroquery/solarsystem/pds/tests/test_ringnode.py

```diff
@@ -0,0 +1,194 @@
+            3098.514,
+        ],
+        [
+            mab["RA (deg)"].to(u.deg).value,
+            mab["Dec (deg)"].to(u.deg).value,
+            mab["dRA"].to(u.arcsec).value,
+            mab["dDec"].to(u.arcsec).value,
+            mab["sub_obs_lon"].to(u.deg).value,
+            mab["sub_obs_lat"].to(u.deg).value,
+            mab["sub_sun_lon"].to(u.deg).value,
+            mab["sub_sun_lat"].to(u.deg).value,
+            mab["phase"].to(u.deg).value,
+            mab["distance"].to(u.km * 1e6).value,
+        ],
+        rtol=1e-2,
+    )
+
+    # check a ring in ringtable
+    beta = ringtable[ringtable.loc_indices["Beta"]]
+    assert np.isclose(beta["pericenter"].to(u.deg).value, 231.051, rtol=1e-3)
+    assert np.isclose(beta["ascending node"].to(u.deg).value, 353.6, rtol=1e-2)
+
+
+def test_ephemeris_query_payload():
+    res = pds.RingNode().ephemeris(
+        planet="Neptune",
+        obs_time="2022-05-03 00:00",
+        neptune_arcmodel=1,
+        location=(10.0 * u.deg, -120.355 * u.deg, 1000 * u.m),
+        get_query_payload=True,
+    )
+
+    assert res == OrderedDict(
+        [
+            ("abbrev", "nep"),
+            ("ephem", "000 NEP081 + NEP095 + DE440"),
+            (
+                "time",
+                "2022-05-03 00:00",
+            ),  # UTC. this should be enforced when checking inputs
+            ("fov", 10),  # next few are figure options, can be hardcoded and ignored
+            ("fov_unit", "Neptune radii"),
+            ("center", "body"),
+            ("center_body", "Neptune"),
+            ("center_ansa", "Adams Ring"),
+            ("center_ew", "east"),
+            ("center_ra", ""),
+            ("center_ra_type", "hours"),
+            ("center_dec", ""),
+            ("center_star", ""),
+            ("viewpoint", "latlon"),
+            (
+                "observatory",
+                "Earth's center",
+            ), 
+            ("latitude", 10),
+            ("longitude", -120.355),
+            ("lon_dir", "east"),
+            ("altitude", 1000),
+            ("moons", "814 All inner moons (N1-N8,N14)"),
+            ("rings", "Galle, LeVerrier, Arago, Adams"),
+            ("arcmodel", "#1 (820.1194 deg/day)"),
+            (
+                "extra_ra",
+                "",
+            ),  # figure options below this line, can all be hardcoded and ignored
+            ("extra_ra_type", "hours"),
+            ("extra_dec", ""),
+            ("extra_name", ""),
+            ("title", ""),
+            ("labels", "Small (6 points)"),
+            ("moonpts", "0"),
+            ("blank", "No"),
+            ("opacity", "Transparent"),
+            ("peris", "None"),
+            ("peripts", "4"),
+            ("arcpts", "4"),
+            ("meridians", "Yes"),
+            ("output", "html"),
+        ]
+    )
+
+
+def test_bad_query_exception_throw():
+
+    with pytest.raises(ValueError):
+        pds.RingNode().ephemeris(planet="Mercury", obs_time="2022-05-03 00:00")
```

Comment:
Here and elsewhere: `RingNode` is already an instance of `RingNodeClass`, so there is no need for the parentheses.  Just `pds.RingNode.ephemeris(...)` will suffice.

Reply:
oh ok, gotcha - now I see what you meant before.  will change